scanpipe/pipelines/deploy_to_develop.py

```diff
@@ -189,6 +189,16 @@ def map_javascript(self):
         """
         d2d.map_javascript(project=self.project, logger=self.log)
 
+    @group("Elf")
+    def map_elf(self):
+        """Map DWARF paths using similarities of path suffixes."""
+        d2d.map_elfs(project=self.project, logger=self.log)
+
+    @group("Go")
+    def map_go(self):
+        """Map GO file paths using similarities of path suffixes."""
+        d2d.map_go_paths(project=self.project, logger=self.log)
+
```

Comment:
Both of the new steps are not included in the DeployToDevelop.steps class method.
This shows that integration tests are lacking a check for those as the tests are passing and we should have a failure.

Reply:
I see it done now